src/main/java/org/jenkinsci/plugins/workflow/cps/UninstantiatedDescribableWithInterpolation.java

```diff
@@ -0,0 +1,29 @@
+package org.jenkinsci.plugins.workflow.cps;
+
+import org.jenkinsci.plugins.structs.describable.UninstantiatedDescribable;
+
+import java.util.Map;
+import java.util.Set;
+
+public class UninstantiatedDescribableWithInterpolation extends UninstantiatedDescribable {
+    private final Set<String> interpolatedStrings;
+
+    public UninstantiatedDescribableWithInterpolation(String symbol, String klass, Map<String, ?> arguments, Set<String> interpolatedStrings) {
+        super(symbol, klass, arguments);
+        this.interpolatedStrings = interpolatedStrings;
+    }
+
+    public Set<String> getInterpolatedStrings() {
+        return interpolatedStrings;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        return super.equals(o);
+    }
+
+    @Override
+    public int hashCode() {
+        return super.hashCode();
+    }
```

Comment:
You could omit these right? Or did you just want to make it clear that `interpolatedStrings` was not part of the equality check? Should `interpolatedStrings` be part of the equality check? I'm not sure if it matters.

Reply:
In this case, it was spotbugs complaining to me that I had no overriden `equals` and `hashcode`